app/controllers/forms_controller.rb

```diff
@@ -0,0 +1,39 @@
+class FormsController < ApplicationController
+  def index
+    @forms = Form.page(params[:page]).per Settings.page_limit
+  end
+
+  def new
+    @form = Form.new
+  end
+
+  def create
+    @form = current_user.forms.build
+    @form.intent = params[:form][:intent]
+    @form.room_id = params[:form][:room_id]
+    @form.time_id = params[:form][:time_id]
+    @form.phone = params[:form][:phone]
+
+    if @form.save
+      flash[:success] = t ".controller.success"
+      redirect_to @form
+    else
+      flash[:danger] = t ".controller.danger"
+      render :new
+    end
+  end
+
+  def show
+    @forms = Form.find_by id: params[:id]
```

Comment:
hàm find chỉ trả ra 1 bản ghi thôi nên chú ý cách đặt tên biến
```suggestion
    @form = Form.find_by id: params[:id]
```